jbmc/src/java_bytecode/ci_lazy_methods.cpp

```diff
@@ -319,7 +319,7 @@ ci_lazy_methodst::convert_and_analyze_method(
   symbol_tablet &symbol_table,
   std::unordered_set<irep_idt> &methods_to_convert_later,
   std::unordered_set<irep_idt> &instantiated_classes,
-  std::unordered_set<exprt, irep_hash> &virtual_function_calls)
+  std::unordered_set<virtual_function_exprt, irep_hash> &virtual_functions)
```

Comment:
`called_virtual_functions`, to match the naming style of `instantiated_classes`?